Missing browserify-css when installing aframe as a dependency

Steps to reproduce:

```bash
mkdir test && cd test
npm install aframe
echo 'require("aframe")' > index.js
browserify index.js -o bundle.js
```

Output:
```
Error: Cannot find module 'browserify-css' from '/Users/donmccurdy/Desktop/test/node_modules/aframe'
    at /Users/donmccurdy/.nvm/versions/node/v5.5.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:46:17
    at process (/Users/donmccurdy/.nvm/versions/node/v5.5.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/donmccurdy/.nvm/versions/node/v5.5.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/donmccurdy/.nvm/versions/node/v5.5.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/donmccurdy/.nvm/versions/node/v5.5.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /Users/donmccurdy/.nvm/versions/node/v5.5.0/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
```

- A-Frame Version: v0.3.0
- Platform/Device: OS X

--- a/package.json
+++ b/package.json
@@ -29,12 +29,13 @@
   "repository": "aframevr/aframe",
   "license": "MIT",
   "dependencies": {
+    "browserify-css": "^0.8.2",
     "debug": "^2.2.0",
     "deep-assign": "^2.0.0",
     "document-register-element": "dmarcos/document-register-element#8ccc532b7",
-    "promise-polyfill": "^3.1.0",
     "object-assign": "^4.0.1",
     "present": "0.0.6",
+    "promise-polyfill": "^3.1.0",
     "style-attr": "^1.0.2",
     "three": "^0.76.1",
     "tween.js": "^15.0.0",
@@ -42,7 +43,6 @@
   },
   "devDependencies": {
     "browserify": "^13.1.0",
-    "browserify-css": "^0.8.2",
     "browserify-derequire": "^0.9.4",
     "browserify-istanbul": "^2.0.0",
     "budo": "^8.1.0",

